src/Action/NotFoundAction.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Action;
+
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+/**
+ * Not found action.
```

Comment:
This summary adds nothing and should be removed, or reworded?

Reply:
* Action that throws a NotFoundHttpException, useful to disable an operation.
 ?

Reply:
An action which always returns HTTP 404 Not Found. Useful for disabling an operation.

(Implementation details should be left out.)